src/platform/Linux/PlatformManagerImpl.cpp

```diff
@@ -402,6 +402,38 @@ CHIP_ERROR PlatformManagerImpl::_GetBootReasons(uint8_t & bootReasons)
     return err;
 }
 
+CHIP_ERROR PlatformManagerImpl::_GetActiveHardwareFaults(GeneralFaults<kMaxHardwareFaults> & hardwareFaults)
+{
+    // On Linux Simulation, set following hardware faults statically.
```

Comment:
This is not only used in tests, so I think it should be guarded with some `#if` to only enable it for testing.